python/F3DPythonBindings.cxx

```diff
@@ -103,6 +106,34 @@ PYBIND11_MODULE(f3d, module)
     .def(
       "getFileInfo", &f3d::loader::getFileInfo, "Get the file index, path, and information string");
 
+  // f3d::camera
+  py::class_<f3d::camera, std::unique_ptr<f3d::camera, py::nodelete> > camera(module, "camera");
+
+  camera.def("setPosition", &f3d::camera::setPosition)
+    .def("getPosition", py::overload_cast<>(&f3d::camera::getPosition))
+    .def("getPosition", py::overload_cast<f3d::point3_t&>(&f3d::camera::getPosition))
```

Comment:
Don't you have to bind the types too?

Reply:
Not sure what you mean, not a pybind11 expert here.

Reply:
ok, got it now.